hledger/Hledger/Cli/Commands/Run.txt

```diff
@@ -44,6 +44,14 @@ Caveats:
 
 -   Numeric flags like -3 do not work, use long form --depth 3
 
+-   You can pass -f to the run itself, and also to any commands given
+    after it (or in the command file, or via REPL). When specific
+    command does not have -f in its flags, it will use the journal(s)
+    specified in the arguments of run. If command does have -f flag,
+    this journal would be read and its contents would be cache, so if
+    several commands specify the same -f flag, they will read the
+    journal only once.
+
```

Comment:
Typo: "cache". 
Also the wording seems to suggest caching is a special thing that happens when command has -f. Perhaps it could be clarified that all input files are cached.
(I sometimes say "input files", since they're not always .journals).